File: dotnet/cafdemalihapa/Directories/Hosting/Feature/Automations/Commands/RunSecondaryApplication.cs
namespace cafdemalihapa.Directories.Hosting.Feature.Automations.Commands
{
    public class RunSecondaryApplication
    (
        CommandsDirectory commandsDirectory
    )
    {
        public string GetPath()
        {
            var commandsDirectoryPath = commandsDirectory.GetPath();
            var operationsDirectory = Path.Combine(commandsDirectoryPath, "run-secondary-application.ps1");
            return operationsDirectory;
        }
    }
}

File: dotnet/cafdemvers/zzq-cafdemwimu/cafdemwimu.console/Directories/Hosting/Feature/Automations/Commands/RunSecondaryApplication.fs
namespace cafdemwimu.console.Directories.Hosting.Feature.Automations.Commands

open System.IO

type RunSecondaryApplication(commandsDirectory: CommandsDirectory) =
    member _.GetPath() =
        let commandsDirectoryPath = commandsDirectory.GetPath()
        let operationsDirectory = Path.Combine(commandsDirectoryPath, "run-secondary-application.ps1")
        operationsDirectory
